api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -342,6 +342,13 @@ message Bootstrap {
   // TODO(abeyad): Add public-facing documentation.
   // [#not-implemented-hide:]
   core.v3.TypedExtensionConfig xds_delegate_extension = 35;
+
+  // Optional XdsConfigTracker configuration, which allows tracking xDS responses in external components,
+  // e.g., external tracer or monitor. It provides the process point when receive, ingest, or fail to
+  // process xDS resources and messages.
+  // If a value is not specified, no XdsConfigTracker will be used.
+  // [#not-implemented-hide:]
```

Comment:
Why is this not-implemented-hide? Isn't it implemented? Also, can you please add a release note? Thank you.

/wait

Reply:
Thanks! This PR only added an api extension interface right now, and there is no actual extension implementation. This enables the ability to extend this interface for different use case. Should I add a release note only for this interface?

Reply:
The fact that it can be implemented means it's implemented, so I would remove the not-implemented-hide and still add a release note for it. It would be also good to `:repo:` link to the test extension so people can look at an example of the API, and I would also mention there are no in-repo extensions currently, etc.

Reply:
Thanks! Done, and waiting CI.